content/components/banner.mdx

```diff
@@ -1,15 +1,14 @@
 ---
 title: Banner
-description: Use Banner to highlight important information.
-railsIds:
+description: Banner is used to highlight important information.
+componentId: banner
```

Comment:
Is this used anywhere?

Reply:
Info from the component metadata repo: https://github.com/primer/component-metadata#primer-component-metadata